src/main/scala/com/fulcrumgenomics/commons/io/Io.scala

```diff
@@ -182,9 +182,14 @@ trait IoUtil {
 
   /** Finds a resource with a given name on the classpath and produces an iterator of lines of text from the resource. */
   def readLinesFromResource(name: String): Iterator[String] = {
-    val in = getClass.getResourceAsStream(name)
-    require(in != null, s"Resource does not exist at path: $name")
-    val stream = new BufferedInputStream(in, bufferSize)
-    Source.fromInputStream(stream).withClose(() => stream.close()).getLines
+    Seq(getClass.getResourceAsStream _, getClass.getClassLoader.getResourceAsStream _)
+      .flatMap(m => Option(m(name)))
+      .headOption match {
+      case None     =>
+        throw new IllegalArgumentException(s"Resource does not exist at path: $name")
+      case Some(in) =>
+        val stream = new BufferedInputStream(in, bufferSize)
+        Source.fromInputStream(stream).withClose(() => stream.close()).getLines
```

Comment:
While we're here, lets change `stream.close()` to `stream.safelyClose()` since it is an input stream?